src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -271,7 +271,7 @@ def test_acr_create_replication(self, resource_group, resource_group_location):
     @ResourceGroupPreparer()
     @record_only()
     def test_acr_image_import(self, resource_group):
```

Comment:
Move this test to a new file called something like `test_acr_import_commands.py` so you don't have to rerecord everything whenever you make changes.

Reply:
Never mind about this one, it seems that the test utils already took care of not rerecording unchanged tests.

Reply:
Which test utils are you talking about? Is it a built in feature of travis-ci?

Reply:
in the latest/recordings/ folder are the yaml test recordings. These are per individual test (method) in each unittest test-case (class). So there is no need to move a test to a separate folder for recording.

Also to re-record a test or multiple test, just run the test or test-case live using the `--live` option with azdev. The test would be run live if the recording is missing or --live is used.